### DISCUSSION TITLE
How to add custom button to open notes menu like "recent changes"?

### DISCUSSION THREAD

**Discussion by user9931** [TriliumNext/Trilium#2196] [Q&A]
Is it possible to create a menu (list of notes) like one that shows on "Recent Changes" button? May be nice way to organize "shortcuts" for  favorite notes and and dont clutter the limited space on the top panel.

**And side question may be "how to hide text on buttons, show only icons" but its probably better for issues then discussions..**

**Comment by zadam** [comment]
Hi, very soon to be released 0.48 version changes completely the design and the top bar is mostly moved to the left side and contains icons only without text.

There is a way to add custom buttons with a script note:

```javascript
api.addButtonToToolbar({
    title: 'My note',
    icon: 'calendar-alt', // box icon name
    action: () => api.activateNote('0I9beeqiAQBU') // argument is target noteId
});
```

Some details are here: https://github.com/zadam/trilium/wiki/Scripts

However this is quite cumbersome and some better support is planned, but not implemented yet.

**Comment by user9931** [comment]
Thank you for your answer, i'm already using couple of buttons (Today + one of my favourite notes), but space are limited so I'm asking about some kind of menu like in "Recent changes"

![image](https://user-images.githubusercontent.com/62539068/135797145-d8c80ca3-66d1-4820-8007-b082f462663f.png)

**Reply by zadam** [reply]
I see. I've been thinking about a button type which would function sort of like a bookmark folder. I take this as quite high priority, but I wasn't yet able to decide on how to model this in the best way.
